src/components/socialMedia/SocialMedia.js

```diff
@@ -62,6 +62,22 @@ export default function socialMedia() {
       :
       null}
 
+      {socialMediaLinks.medium ? 
+      (<a href={socialMediaLinks.medium} className="icon-button medium" target="_blank">
```

Comment:
Add `medium` class styling to [SocialMedia.css](https://github.com/saadpasta/developerFolio/blob/master/src/components/socialMedia/SocialMedia.css)

Reply:
@kartikcho ahhh good catch I did miss this in my diff to my fork seen at https://bradleycherrin.com

done.